src/Symfony/Component/Messenger/Tests/Transport/AmqpExt/AmqpExtIntegrationTest.php

```diff
@@ -75,6 +75,10 @@ public function testItSendsAndReceivesMessages()
         $this->assertEmpty(iterator_to_array($receiver->get()));
     }
 
+    /**
+     * @group legacy
+     * ^ for now, deprecation errors are thrown during serialization.
```

Comment:
how can people prevent the deprecation from happening?

Reply:
Currently, the deprecation is being triggered by a fallback. This fallback is used in case people have messages in the queue while updating their codebase to a version that includes this PR.
Newer messages that are retried after this PR is downloaded will no longer trigger this deprecation.